tools/tracegen/TDFParser.cpp

```diff
@@ -365,7 +365,7 @@ TDFParser::getTemplate(const char *line, const char *key)
 	}
 
 	len = (pos - vpos);
-	for (int i = strlen(line); i >= 0; i--){
+	for (size_t i = strlen(line); i >= 0; i--){
 		if (line[i] == ' '){
```

Comment:
Since this is just doing a continue can we not remove it and change the next `else if` to an `if`?